src/components/editTestCase/qiCore/LeftPanel/ElementsTab/builder/element/TypeEditor.tsx

```diff
@@ -101,6 +102,30 @@ const TypeEditor = ({
             value={value}
           />
         );
+      case "positiveInt":
+        return (
+          <IntegerComponent
+            canEdit={true}
+            structureDefinition={structureDefinition}
+            fieldRequired={required}
+            label={``}
+            onChange={onChange}
+            value={value}
+            signed={false}
+          />
+        );
+      case "unsignedInt":
+        return (
+          <IntegerComponent
+            canEdit={true}
+            structureDefinition={structureDefinition}
+            fieldRequired={required}
+            label={`Integer Field`}
+            onChange={onChange}
+            value={value}
+            signed={true}
```

Comment:
if this is for "unsignedInt" then shouldn't signed be false?
Looks like the main difference between unsignedInt and positiveInt is that unsignedInt includes 0, and positiveInt does not

Reply:
@nmorasb yeah you are right I'll make the change.